assistant/v1.ts

```diff
@@ -5198,9 +5336,11 @@ namespace AssistantV1 {
     agent_unavailable?: AgentAvailabilityMessage;
     /** Routing or other contextual information to be used by target service desk systems. */
     transfer_info?: DialogNodeOutputConnectToAgentTransferInfo;
+    /** An array of objects specifying channels for which the response is intended. */
+    channels?: ResponseGenericChannel[];
   }
 
-  /** An object that describes a response with response type `image`. */
+  /** DialogNodeOutputGenericDialogNodeOutputResponseTypeImage. */
```

Comment:
It's interesting that we're seeing these changes from an actual description to what looks like an ID. It doesn't seem very desirable to me. This is coming from the generator, right? I wonder if the descriptions were removed from the API definition.

It only seems to be happening in Assistant (v1 and v2)

Reply:
Yes looking at the previous version of the api refs it looks like the description for these discriminator models was removed. As to why, I would not know but it had to be done on purpose

Reply:
Yeah that's weird. Nothing that should hold up the release though